app/schemas/USER.json

```diff
@@ -0,0 +1,68 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "properties": {
+      "userId": {
+        "type": "string"
+      },
+      "f_name": {
```

Comment:
Pedantic nit-pick: I think f_name and l_name would be nicer as `first_name` and `last_name`.